packages/create-cloudflare/src/common.ts

```diff
@@ -361,6 +375,26 @@ async function isGitInstalled() {
 	return (await getGitVersion()) !== null;
 }
 
+async function isGitConfigured() {
+	try {
+		const userName = await runCommand("git config user.name", {
+			useSpinner: false,
+			silent: true,
+		});
+		if (!userName) return false;
+
+		const email = await runCommand("git config user.name", {
+			useSpinner: false,
+			silent: true,
+		});
+		if (!email) return false;
+
+		return true;
+	} catch {
+		return null;
```

Comment:
nit: why not `return false` here? below 👇 in the catch you do return `false`, so for consistency I guess I'd do the same here? (or return `null` there too)